ntbs-service/Models/NotificationGroup.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace ntbs_service.Models
+{
+    public class NotificationGroup

```

Comment:
Can you add a comment explaining the real-world understanding of what a group is? It's not super clear from the term we've given it, so worth explaining I think :)